submissions/finalise.R

```diff
@@ -21,17 +67,21 @@ submission <- submission[, value := as.integer(value)]
 
 # # Replace some states with a single model ---------------------------------
 source(here("utils", "get-locations.R"))
-swap <- list(
-  "Case convolution" = c("Delaware", "Ohio", "Rhode Island", "Virginia", "Utah", "Hawaii", "Maine"),
-  "Rt" = c("Arkansas", "Colorado", "Columbia")
-)
-if (length(swap) > 0) {
-  for(swap_model in names(swap)) {
+if (file.exists(here("submissions", "utils", paste0(target_date, "-swap-ensemble.rds")))) {
+  swap <- readRDS(here("submissions", "utils", paste0(target_date, "-swap-ensemble.rds")))
+  
+  for (swap_model in names(swap)) {
     swap_names <- data.frame("state" = swap[swap_model][[1]])
     swap_locs <- merge(swap_names, state_locations, by = "state")$location
-    alt_subs <- load_submissions(target_date, "all-models", summarise = FALSE) 
+    if ("US" %in% swap_names$state) {
+      swap_locs <- c(swap_locs, "US")
+    }
```

Comment:
again maybe if possible call this section as a function (not required but might keep the script easier to understand.